### PR TITLE
gphoto2: update to 2.5.28

### PR DESCRIPTION
**Pull request by kmilos**

### CODE REVIEW

File: mingw-w64-gphoto2/PKGBUILD [hunk 1]
```diff
@@ -18,17 +18,13 @@ depends=("${MINGW_PACKAGE_PREFIX}-libgphoto2"
 makedepends=("${MINGW_PACKAGE_PREFIX}-autotools")
 validpgpkeys=('7C4AFD61D8AAE7570796A5172209D6902F969C95') # Marcus Meissner
 source=("https://github.com/gphoto/gphoto2/releases/download/v${pkgver}/${_realname}-${pkgver}.tar.xz"
```
Comment (Biswa96) [inline mingw-w64-gphoto2/PKGBUILD:20]:
```suggestion
source=("https://github.com/gphoto/gphoto2/releases/download/v${pkgver}/${_realname}-${pkgver}.tar.xz"{,.asc}
```

File: mingw-w64-libgphoto2/PKGBUILD [hunk 2]
```diff
@@ -20,15 +20,15 @@ depends=("${MINGW_PACKAGE_PREFIX}-libsystre"
          "${MINGW_PACKAGE_PREFIX}-libltdl")
 makedepends=("${MINGW_PACKAGE_PREFIX}-autotools"
              "${MINGW_PACKAGE_PREFIX}-doxygen")
-source=(https://github.com/gphoto/libgphoto2/releases/download/v${pkgver}/${_realname}-${pkgver}.tar.xz
-        libgphoto2-gp_system_filename-fix.patch
-        libgphoto2-fix-lumix.patch
-	libgphoto2-configure-ws232.patch)
-sha256sums=('a7253bd18c6ef65352150ac36a668f883a5ad2634eb6455b3ca9aa6ac98d0073'
+source=("https://github.com/gphoto/libgphoto2/releases/download/v${pkgver}/${_realname}-${pkgver}.tar.xz"
```
Comment (Biswa96) [inline mingw-w64-libgphoto2/PKGBUILD:23]:
```suggestion
source=("https://github.com/gphoto/libgphoto2/releases/download/v${pkgver}/${_realname}-${pkgver}.tar.xz"{,.asc}
```

**Comment by Biswa96** [comment]
In previous libgphoto2 commit, `-Wl, -rpath` option was removed for clang. Would it be same as adding `--disable-rpath` option with configure command instead of that sed command? /cc @raedrizqie

**Comment by kmilos** [comment]
Thanks, I didn't change that since the last version... Do you know if the the signature is in the db?

**Comment by Biswa96** [comment]
> Do you know if the the signature is in the db?

Yup.

```
$ /usr/bin/gpg --recv-keys 7C4AFD61D8AAE7570796A5172209D6902F969C95
gpg: key 2209D6902F969C95: public key "Marcus Meißner (german spelling) <meissner@suse.de>" imported
gpg: Total number processed: 1
gpg:               imported: 1
```

**Comment by raedrizqie** [comment]
> In previous libgphoto2 commit, `-Wl, -rpath` option was removed for clang. Would it be same as adding `--disable-rpath` option with configure command instead of that sed command? /cc @raedrizqie

i tried using `--disable-rpath` .. clang still failed the same

**Pull request merged**